bin/01_download_images.py

```diff
@@ -28,28 +34,23 @@
     type=int,
     default=15,
 )
-parser.add_argument(
-    "-c",
-    "--classify",
-    type=io.str2bool,
-    nargs='?',
-    const=True,
-    default=False,
-    help="Classify downloaded images, and sort them to training set.",
-)
 args = parser.parse_args()
 logging.info(f"args.n_images: {args.n_images}")
 logging.info(f"args.sleep: {args.sleep}")
-logging.info(f"args.classify: {args.classify}")
 
 
-paths.create_paths_if_not_exist()
-
-
-if args.classify:
-    model = load_model(str(paths.CLASSIFIER_HDF5))
+# download image classifier (if not exists), and load to memory
+if not paths.CLASSIFIER_HDF5.exists():
+    wget.download(
+        url="https://github.com/codeformuenster/traffic-cam-data/blob/master/model/model.hdf5?raw=true",
+        out=str(paths.CLASSIFIER_HDF5),
+    )
+model = load_model(str(paths.CLASSIFIER_HDF5))
 
+# prepare paths to write to
+paths.create_paths_if_not_exist()
```

Comment:
Do it before, otherwise it will fail if parent directory "data/classifier" does not exist.

```
# prepare paths to write to
paths.create_paths_if_not_exist()

# download image classifier (if not exists), and load to memory
if not paths.CLASSIFIER_HDF5.exists():
    wget.download(
        url="https://github.com/codeformuenster/traffic-cam-data/blob/master/model/model.hdf5?raw=true",
        out=str(paths.CLASSIFIER_HDF5),
    )
model = load_model(str(paths.CLASSIFIER_HDF5))
```